packages/nextjs/services/plausible.ts

```diff
@@ -10,11 +10,13 @@ export async function trackPlausibleEvent(
   eventName: PlausibleEvent,
   props?: Record<string, string | number | boolean>,
   request?: Request,
+  referrer?: string | null,
 ) {
   const payload = {
     domain: "speedrunethereum.com",
     name: eventName,
     url: `https://speedrunethereum.com`,
+    referrer: referrer || "",
```

Comment:
Check https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/221/files#r2193042006

It seems that this is allowed: https://plausible.io/docs/events-api